.gitignore

```diff
@@ -1 +1,11 @@
 bfvmm/bin/cross/libc++.so
+.build/
+*.a
+bin/
```

Comment:
Being that they are already indexed, and boilerplate, it's not an issue. Having said that, I can add an exception to the gitignore for Makefiles